src/pages/imprint.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { HelmetSeo } from '~/components/Helmet/HelmetSeo';
+import { LayoutArticle } from '~/components/Layout';
+import { ExternalLink } from '~/components/Links/ExternalLink';
+import BmdvFundingImage from '~/static/BMDV_Foerderung.svg';
+
+const ImprintPage = () => {
+  return (
+    <LayoutArticle>
+      <HelmetSeo noindex title="Impressum" />
+      <h2>Impressum</h2>
+      <p>
+        <b>FixMyCity GmbH</b>
+        <br />
+        Karlsgartenstraße 12
+        <br />
+        12049 Berlin
+        <br />
+        E-Mail-Adresse:{' '}
+        <a href="mailto:hello@fixmycity.de">hello@fixmycity.de</a>
```

Comment:
```suggestion
        <ExternalLink href="mailto:hello@fixmycity.de">hello@fixmycity.de</ExternalLink>
```

Which will apply the link styles. We could also create a `MailtoLink` component with possible addition of `subject`, `body` probs in the future.